nipype/pipeline/plugins/multiproc.py

```diff
@@ -217,32 +178,38 @@ def _send_procs_to_workers(self, updatehash=False, graph=None):
         executing_now = []
 
         # Check to see if a job is available
-        currently_running_jobids = np.flatnonzero((self.proc_pending == True) & \
-                                (self.depidx.sum(axis=0) == 0).__array__())
+        currently_running_jobids = np.flatnonzero(
+            self.proc_pending & (self.depidx.sum(axis=0) == 0).__array__())
 
         # Check available system resources by summing all threads and memory used
         busy_memory_gb = 0
         busy_processors = 0
         for jobid in currently_running_jobids:
-            if self.procs[jobid]._interface.estimated_memory_gb <= self.memory_gb and \
-                            self.procs[jobid]._interface.num_threads <= self.processors:
-
-                busy_memory_gb += self.procs[jobid]._interface.estimated_memory_gb
-                busy_processors += self.procs[jobid]._interface.num_threads
-
-            else:
-                raise ValueError(
-                    "Resources required by jobid {0} ({3}GB, {4} threads) exceed what is "
-                    "available on the system ({1}GB, {2} threads)".format(
-                        jobid, self.memory_gb, self.processors,
-                        self.procs[jobid]._interface.estimated_memory_gb,
-                        self.procs[jobid]._interface.num_threads))
+            est_mem_gb = self.procs[jobid]._interface.estimated_memory_gb
+            est_num_th = self.procs[jobid]._interface.num_threads
+
+            if est_mem_gb > self.memory_gb:
+                logger.warning(
+                    'Job %s - Estimated memory (%0.2fGB) exceeds the total amount'
+                    ' available (%0.2fGB).', self.procs[jobid].name, est_mem_gb, self.memory_gb)
+                if self.raise_insufficient:
+                    raise RuntimeError('Insufficient resources available for job')
+
+            if est_num_th > self.processors:
+                logger.warning(
+                    'Job %s - Requested %d threads, but only %d are available.',
+                    self.procs[jobid].name, est_num_th, self.processors)
+                if self.raise_insufficient:
+                    raise RuntimeError('Insufficient resources available for job')
+
+            busy_memory_gb += min(est_mem_gb, self.memory_gb)
+            busy_processors += min(est_num_th, self.processors)
 
         free_memory_gb = self.memory_gb - busy_memory_gb
         free_processors = self.processors - busy_processors
 
         # Check all jobs without dependency not run
-        jobids = np.flatnonzero((self.proc_done == False) & \
+        jobids = np.flatnonzero((self.proc_done == False) &
```

Comment:
`~self.proc_done`?

Reply:
`self.proc_done` is not a numpy array

Reply:
I was wrong. Changing to `~self.proc_done`